### PR TITLE
UI Update

### PR DESCRIPTION
**Pull request by kevindmoore**

### CODE REVIEW

File: 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart [hunk 1]
```diff
@@ -139,7 +140,7 @@ class _RecipeListState extends State<RecipeList> {
         child: Row(
           children: [
             IconButton(
-              icon: Icon(Icons.search),
+              icon: Icon(Icons.search,),
```
Comment (funkyboy) [inline 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Is this a typo? The same is in the starter project

**Review by funkyboy — COMMENTED** [review]
Left a comment.

File: 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart (hunk 1 above)
Comment (kevindmoore) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Nope. Just a few differences in my final project and these projects. Trying to get them to be the same

Comment (funkyboy) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Ok but is this comma needed?
```
Icons.search,),
            ^
```

Comment (kevindmoore) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: No. Just common to add commas after everything for formatting

Comment (kevindmoore) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Probably should remove it, but then would have to remove it everywhere

Comment (funkyboy) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Oh ok. Makes sense. Let's keep it as is.

**Review by funkyboy — APPROVED** [review]

File: 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart (hunk 1 above)
Comment (Stef-GMS) [reply 11-serialization-with-json/projects/final/lib/ui/recipes/recipe_list.dart:143]: Based on my convo with @orionthewake the formatting comma should be included. This puts the code in line with how Google formats the code.

**Pull request merged**